clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangelogConsumerImpl.java

```diff
@@ -1002,39 +1002,53 @@ protected ChangelogClientConfig getChangelogClientConfig() {
     return changelogClientConfig;
   }
 
-  private class HeartbeatReporterThread extends Thread {
-    HeartbeatReporterThread() {
+  protected HeartbeatReporterThread getHeartbeatReporterThread() {
+    return heartbeatReporterThread;
+  }
+
+  protected BasicConsumerStats getChangeCaptureStats() {
```

Comment:
Minor: Seems not useful here, as `HeartbeatReporterThread` directly use  `changeCaptureStats`.

Reply:
It's just for testing to get code coverage up.

Reply:
Seems not used in test.